ynput/OpenPype
# Traypublisher & Chore: Instance version on follow workfile version

## Changelog Description
If `follow_workfile_version` is enabled but context does not have filled workfile version, a version on instance is used instead.

## Additional info
TrayPublisher does not have any workfile version and it is not possible to define the follow version value per host. So if workfile version is not set it is silently changed to use intance version.

## Testing notes:
1. Enable `Allow version control` in traypublisher simple creator
2. Enable `follow_workfile_version` in `global/publish/CollectAnatomyInstanceData/follow_workfile_version`
3. Open traypublisher
4. Create instance and change the version
5. Publish
6. The custom version should be used

Resolves https://github.com/ynput/OpenPype/issues/5513
